src/plugins/data/public/plugin.ts

```diff
@@ -100,6 +102,10 @@ export class DataPublicPlugin implements Plugin<DataPublicPluginSetup, DataPubli
 
     expressions.registerFunction(esaggs);
 
+    const getInternalStartServices: getInternalStartServicesFn = () => ({
+      fieldFormats: getFieldFormats(),
```

Comment:
In the future, we can remove `getters/setters` altogether, but now using this approach we can save support of two ways for passing dependencies and refactor all places without any rush.